packages/mui-joy/src/Select/Select.tsx

```diff
@@ -500,7 +501,6 @@ const Select = React.forwardRef(function Select<TValue extends {}>(
     additionalProps: {
       ref: listboxRef,
       anchorEl,
-      disablePortal: true,
```

Comment:
I don't recall why `disablePortal` is used here but I think it should be consistent with other popup.